exonum/src/runtime/dispatcher/schema.rs

```diff
@@ -135,7 +135,7 @@ impl<T: Access> Schema<T> {
         self.access.clone().get_map(PENDING_INSTANCES)
     }
 
-    fn local_migration_results(&self) -> MapIndex<T::Base, str, MigrationStatus> {
+    pub(crate) fn local_migration_results(&self) -> MapIndex<T::Base, str, MigrationStatus> {
```

Comment:
Nit / subjective: IMO, `remove_local_migration_result` can be placed in this module; thus, creating a separate module for a single method would be avoided 🙃